### DISCUSSION TITLE
服务器安装自签证书后依旧出现“证书错误”

### DISCUSSION THREAD

**Discussion by wukibaka** [1715173329/UnblockNeteaseMusic#454] [Q&A]
![Screenshot_20220111-182058.jpg](https://user-images.githubusercontent.com/51240031/148925161-440ef1f6-872a-4c60-ab6e-1969a18575cd.jpg)
![Screenshot_20220111-182055.jpg](https://user-images.githubusercontent.com/51240031/148925171-3549560c-20dd-44d7-8e90-a1ea98ab057b.jpg)

**Comment by 1715173329** [comment]
请安装为系统证书。

**Reply by pan93412** [reply]
感覺他好像沒有攔截成功？HTTPS 憑證不對。

**Reply by 1715173329** [reply]
他应该是自己生成了证书吧。

**Reply by pan93412** [reply]
> 为使用客户端是系统安装证书？

？

**Reply by wukibaka** [reply]
Debian怎么把证书安装为系统证书？
我用网上的方法装的

**Reply by 1715173329** [reply]
需要在你的客户端安装证书
